mathosphere-core/src/main/java/com/formulasearchengine/mathosphere/mathpd/FlinkPd.java

```diff
@@ -93,5 +77,19 @@ public WikiDocumentOutput outDocFromText(FlinkPdCommandConfig config, String inp
         return env.readFile(inp, config.getRef());
     }
 
+    public String runFromText(FlinkPdCommandConfig config, String input) throws Exception {
```

Comment:
to simplify the review in the future please use a new pull request for code-hygiene changes like rearranging the order of the methods